parachain/pallets/ethereum-light-client/src/mock.rs

```diff
@@ -157,14 +157,17 @@ pub fn genesis_ethereum_block_hash() -> H256 {
 }
 
 pub fn child_of_genesis_ethereum_header() -> EthereumHeader {
-	child_of_header(&genesis_ethereum_header())
+	child_of_header(&genesis_ethereum_header(), None)
 }
 
-pub fn child_of_header(header: &EthereumHeader) -> EthereumHeader {
+pub fn child_of_header(header: &EthereumHeader, timestamp: Option<u64>) -> EthereumHeader {
 	let mut child: EthereumHeader = Default::default();
 	child.difficulty = 1.into();
 	child.parent_hash = header.compute_hash();
 	child.number = header.number + 1;
+	if let Some(timestamp) = timestamp {
+		child.timestamp = timestamp
+	}
```

Comment:
Is this added for test purposes only? I see the timestamps being set in the tests: https://github.com/Snowfork/snowbridge/blob/f5e2ea1c9ad29a61db31b4898e3983cfd3442c46/parachain/pallets/ethereum-light-client/src/tests.rs#L550

What does adding the timestamp accomplish?

Reply:
So I needed two different blocks with the same number to have different hashes. So I needed to pick some field on the header type and alter it in some way. I chose timestamp because to make the tests easier to digest in the sense of two blocks with the same number coming in at different points in time. But I could of chosen any field really.

Reply:
Great, makes sense, thanks for the explanation.